### ISSUE TITLE
Use `.internal` argument with `cli::cli_abort()`

### ISSUE THREAD

**Issue by olivroy** [r-lib/cli#489]
I am developing a package. I am switching my error calls from `rlang::abort()` to `cli::cli_abort()`
```r
abort("Problem", .internal = TRUE)
This is an internal error in the canadr package, please report it to the package authors.
cli::cli_abort("Problem", .internal = TRUE)
This is an internal error in the cli package, please report it to the
  package authors.
```
But in the cli documentation, it is written that the ... are passed from `cli::abort()` to `rlang::abort()`

---
I don't know how to provide a reprex.

**Comment by gaborcsardi** [comment]
@lionel- Is there a workaround here? Setting `.frame` does not seem to help.

**Comment by davidchall** [comment]
@lionel- I've also experienced this issue with the latest rlang.

**Comment by lionel-** [comment]
@davidchall You also need dev cli for a full fix, see #479.

**Comment by gaborcsardi** [comment]
So as I understand this is fixed now, and there is nothing to do in cli?
